Makefile

```diff
@@ -0,0 +1,12 @@
+all: test
+
+clean:
+	find . -name '*.pyc' -exec rm \{\} \;
+
+flake8:
```

Comment:
Sorry, it was not clear.

This `flake8 custom instruction is unnecessarily complex. Just use a regular `flake8`, as done in the country template. Of course we want to run our tests, not run `openfisca-run-test` on a folder that doesn't contain any YAML test.